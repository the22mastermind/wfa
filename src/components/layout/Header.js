import React from 'react';

const Header = () => {
  return (
    <nav>
      <a href="/" className="brand">Weather Forecast App</a>
      <ul className="nav-links">
        <li><a href="/signup" className="link">Sign Up</a></li>
        <li><a href="/signin" className="small-btn">Sign In</a></li>
      </ul>
    </nav>
  );
}

export default Header;
