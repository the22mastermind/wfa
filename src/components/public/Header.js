import React from 'react';

export default function Header() {
  return (
    <nav>
      <a href="/" className="brand">Weather Forecast App</a>
      <ul className="nav-links">
        <li><a href="/signup" className="link">Sign Up</a></li>
        <li><a href="/signin" className="small-btn">Sign In</a></li>
      </ul>
    </nav>
  );
}
