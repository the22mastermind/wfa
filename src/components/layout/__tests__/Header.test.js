import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../Header';


it('matches the Header snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
