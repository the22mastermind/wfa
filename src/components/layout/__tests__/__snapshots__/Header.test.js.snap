// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the Header snapshot 1`] = `
<nav>
  <a
    className="brand"
    href="/"
  >
    Weather Forecast App
  </a>
  <ul
    className="nav-links"
  >
    <li>
      <a
        className="link"
        href="/signup"
      >
        Sign Up
      </a>
    </li>
    <li>
      <a
        className="small-btn"
        href="/signin"
      >
        Sign In
      </a>
    </li>
  </ul>
</nav>
`;
